src/main/xerus/monstercat/downloader/SongView.kt

```diff
@@ -107,11 +98,51 @@ class SongView(private val sorter: ObservableValue<ReleaseSorting>):
 			}
 		}
 		
-		val defaultItems = {
-			arrayOf(MenuItem("Expand all") { expandAll() },
-				MenuItem("Collapse all") { expandAll(false) })
+		val menuPlay = MenuItem("Play") {
+			val selected = selectionModel.selectedItem ?: return@MenuItem
```

Comment:
shouldn't this also allow multi-selection then?

Reply:
We could have multi selection pretty easily for sure. It's two lines. I am not sure if this wouldn't cause some other issues though.

Reply:
You would have to handle it when asking for the selection, but if you have a right-click menu it might be quite frustrating to not have multi-selection. It's not a focus tho.

Reply:
e75fed79f4d0239a345d27edc541d8ea9bee1227

Reply:
code might be improved by creating an alias / custom getter for ``selectionModel.selectedItems.map{it.value}``